javatests/arcs/showcase/typevariables/BUILD

```diff
@@ -0,0 +1,47 @@
+load(
+    "//third_party/java/arcs/build_defs:build_defs.bzl",
+    "arcs_kt_gen",
+    "arcs_kt_jvm_test_suite",
+)
+
+arcs_kt_gen(
+    name = "var_generation",
+    srcs = ["variable.arcs"],
+)
+
+arcs_kt_jvm_test_suite(
+    name = "variable-tests",
+    srcs = glob(["*Test.kt"]) + [
+        "OrderIngestion.kt",
```

Comment:
Might be worthwhile to split these files off into another `arcs_kt_jvm_library` target?

Reply:
👍